pandas/_libs/tslib.pyx

```diff
@@ -3960,7 +3889,7 @@ for _maybe_method_name in dir(NaTType):
             def f(*args, **kwargs):
                 raise ValueError("NaTType does not support " + func_name)
             f.__name__ = func_name
-            f.__doc__ = _get_docstring(_method_name)
+            f.__doc__ = _get_docstring(func_name)
```

Comment:
This is an unrelated bug.  This block of code exists to make `NaT` raise `ValueError` for each of a bunch of `datetime` methods.  This is supposed to attach the original docstring to the new function `f`.  The typo `_method_name` instead of `func_name` means that a bunch of the docstrings are currently wrong.